src/Services/Transaction.java

```diff
@@ -0,0 +1,59 @@
+package Services;
```

Comment:
is it possible to have a root namespace appended, like `YaYa/Sdk/` or `YaYaSdk` so the package be like `YaYa/Sdk/Service` or `YaYaSdk/Service`?